modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_view_display.node.article.default.yml

```diff
@@ -6,14 +6,17 @@ dependencies:
     - field.field.node.article.field_content
     - field.field.node.article.field_meta_tags
     - field.field.node.article.field_short_title
+    - field.field.node.article.field_social_sharing
     - field.field.node.article.field_subhead
     - field.field.node.article.field_summary
     - field.field.node.article.field_tags
     - field.field.node.article.field_thumbnail
+    - field.field.node.article.field_yoast_seo
```

Comment:
Again, is this correct ?

Reply:
This is added by the yoast_seo module and we remove it. Please help me understand why we need to do this.